coldfront/core/project/views.py

```diff
@@ -269,6 +271,17 @@ def get_context_data(self, **kwargs):
         context['can_request_sec_dir'] = \
             pi_eligible_to_request_secure_dir(self.request.user)
 
+        # show survey responses if available
+        allocation_request = SavioProjectAllocationRequest.objects.filter(
+            project=self.object).first()
+        if allocation_request:
+            context['survey_answers'] = SavioProjectSurveyForm(
+                initial=allocation_request.survey_answers, disable_fields=True)
+
+        allocation_requests = SavioProjectAllocationRequest.objects.filter(project=self.object)
```

Comment:
This `filter` call is being made twice--you can get them all above and then use `.exists()` to check if any exist before iterating over them in the `lambda` below.